web/concrete/single_pages/dashboard/reports/logs.php

```diff
@@ -67,12 +67,18 @@
                     ?></td>
                     <td valign="top"><strong><?=$ent->getType()?></strong></td>
                     <td valign="top"><strong><?php
-                    if($ent->getUserID() == NULL){
+                    $uID = $ent->getUserID();
+                    if(empty($uID)) {
                         echo t("Guest");
                     }
-                    else{
-                        $u = User::getByUserID($ent->getUserID());
-                        echo $u->getUserName();
+                    else {
+                        $u = User::getByUserID($uID);
+                        if(is_object($u)) {
+                            echo $u->getUserName();
+                        }
+                        else {
+                            echo tc('Deleted user', 'Deleted (id: %s)', $uID);
```

Comment:
I mostly asked because the existing contexts we have related to an object / a class in concrete5 but maybe I'm missing something. I assume the translation would be different in most languages, that's why I'd have written "Deleted user.."